killerbee/__init__.py

```diff
@@ -9,7 +9,8 @@
 from kbutils import *      #provides serial, usb, USBVER
 from zigbeedecode import * #would like to import only within killerbee class
 from dot154decode import * #would like to import only within killerbee class
-from config import *       #to get DEV_ENABLE_* variables
+from config import *  # for backwords compatibility
```

Comment:
Is this for other functions used or something? Should we cut these all over and remove the old import?

Reply:
No, but I thought about code of third-party projects which could have used imported the variables. But as I rethink about it, if we want to achieve this, I have to replace e.g. a `config.DEV_ENABLE_SL_NODETEST` with `config.DEV_ENABLE_SL_NODETEST or DEV_ENABLE_SL_NODETEST`to take also account for possible changes of these variables.

Reply:
I added this now to the old variables `DEV_ENABLE_FREAKDUINO`, `DEV_ENABLE_ZIGDUINO`, `DEV_ENABLE_SL_BEEHIVE`, `DEV_ENABLE_SL_NODETEST`. I didn't added this to the `DEV_ENABLE_(TELOSB/APIMOTE2/APIMOTE1), since these variables were never in the master branch (and only since a few weeks in the development branch).